blocks/link/link.css

```diff
@@ -16,4 +16,32 @@
   position:relative;
   left: 0.3rem;
   top: 0.5rem;
+}
+
+.link.block.no-buttons.blue-arrow a::after {
+  content: url("../../icons/link-blue.svg");
+  position: relative;
+  top: 0.3rem;
+}
+
+.link.block.transparent a {
+    background: var(--white);
```

Comment:
Nit: there's double the spacing here

Reply:
If its `transparent` , ideally it should have no color. No?